### PR TITLE
[TASK] Add meta.* property replace

### PR DESCRIPTION
**Pull request by maddy2101**
With #81464, a new property for meta tags was introduced.

### CODE REVIEW

**Comment by marble** [comment]
Can you make the examples meaningful? I have no idea what they try to show. And, please: Never use "foo" and "bar" in examples! ((e.g. Stop using foo bar ... https://dev.to/warrend/no-more-foo-bar-and-baz-56j4))

File: Documentation/Conditions/Reference/Index.rst [hunk 1]
```diff
@@ -1009,3 +1009,40 @@ shows how quotes can be escaped::
...
+.. _condition-site:
+
+site
+====
+
+Syntax::
+
+   [site = identifier = someIdentifier, base = https://www.typo3.org/]
+
+Comparison: The identifier of the site name is evaluated.
```
Comment (DanielSiepmann) [inline Documentation/Conditions/Reference/Index.rst]: What I'm missing is an information about the `base`. What is `base` in this syntax, what are possible values?

File: Documentation/Conditions/Reference/Index.rst [hunk 2]
```diff
@@ -1009,3 +1009,40 @@ shows how quotes can be escaped::
...
+
+
+.. _condition-site-language:
+
+siteLanguage
+============
+
+Syntax::
+
+   [siteLanguage = locale = de_CH.UTF-8, title = Switzerland]
```
Comment (DanielSiepmann) [inline Documentation/Conditions/Reference/Index.rst]: Same as `base` above, what's `title`, is it a kind of sub property?

**Comment by maddy2101** [comment]
will come up with a more complete all-in-all documentation about site configuration. This TS-part can then link there. Please leave this PR open until we delivered.

**Comment by DanielSiepmann** [comment]
Already merged. I took the time to also "refactor" the condition reference. It's now sorted alphabetical and follows the new syntax, see  #159

Would be nice if you can start further improvements from there.

**Pull request merged**